src/routes/auth.route.ts

```diff
@@ -24,8 +24,26 @@ authRouter.route("/register").post(async (req: Request, res: Response) => {
     const data = req.body;
     const result = await register(data);
     res.send(result);
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  } catch (error: any) {
+  } catch (error: unknown) {
+    let message;
+    if (error instanceof Error) {
```

Comment:
@VitaliiDenysyuk affect 🤦 
please check our telegram chat, and use simple `error` without any `error.message`